autogen/main/variables.tf.tmpl

```diff
@@ -614,9 +614,7 @@ variable "node_metadata" {
     error_message = "The node_metadata value must be one of GKE_METADATA, GCE_METADATA, UNSPECIFIED, GKE_METADATA_SERVER or EXPOSE."
   }
 }
-{% endif %}
 
-{% if autopilot_cluster != true %}
```

Comment:
Repeated condition to the preceding one here https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/pull/1539/files#diff-c4d9c31e42e8a940e005adfa20b2243e0284ef6ed49b3884b0324a32770f4f6bR594 so removed